src/loki/search.cc

```diff
@@ -585,6 +585,7 @@ struct bin_handler_t {
               opp_reach.inbound >= p_itr->location.min_inbound_reach_) {
             tile = opp_tile;
             edge = opp_edge;
+            edge_id = opp_edgeid;
```

Comment:
ill review the code after this (finalize function) but this was also not right. when we bail on the main edge and swap in the opposing edge we were forgetting to swap in the id

Reply:
yeah so the only thing the id gets used for in the finalize method is to look up the reach, which get set on the search result but isnt otherwised used later in the request processing. so this fix makes it so that you dont see the wrong reach value in locate for example but otherwise has no impact that i can see